### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by msrose**
Consolidated dev/regular deps and added yvmrc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,28 +4,28 @@
   "description": "commit message status check for continuous integration",
   "main": "index.js",
   "repository": "git@github.com:tophat/commit-watch.git",
-  "author": "Shouvik DCosta <shouvik.dcosta@tophatmonocle.com>",
+  "author": "Top Hat Monocle Corp. <opensource@tophat.com>",
```
Comment (sanchitgera) [inline package.json]: This should be just Top Hat

File: package.json [hunk 2]
```diff
@@ -4,28 +4,28 @@
...
     "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-import": "^2.17.2",
     "eslint-plugin-jest": "^22.4.1",
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-utils": "^1.3.1",
-    "junit-report-builder": "^1.3.2",
-    "lodash": "^4.17.11",
     "prettier": "^1.17.0"
+  },
+  "dependencies": {
```
Comment (sanchitgera) [inline package.json]: Most of these should be dev dependencies

File: package.json (hunk 1 above)
Comment (msrose) [reply package.json]: Why? Was thinking to use the legal name of the company 😛

File: package.json (hunk 2 above)
Comment (msrose) [reply package.json]: As far as I know these are used directly by commit-watch, so I think they have to be dependencies if it's going to be installed correctly.

File: package.json (hunk 1 above)
Comment (sanchitgera) [reply package.json]: Actually I'm not sure. We use different names in different places. YVM uses just Top Hat https://github.com/tophat/yvm/blame/d2953e266b788345bcb084491e806336667b9f56/package.json#L29 but we use Top Hat Monocle everywhere else 🤷‍♂️

File: package.json (hunk 2 above)
Comment (sanchitgera) [reply package.json]: Makes sense. It's a dev dependency on the other projects.

**Pull request merged**